draft/spec/index.html

```diff
@@ -211,8 +211,9 @@ <h2>Terminology</h2>
           directory in a zip file.</dd>
         <dt><dfn>OCFL Storage Root</dfn>:</dt>
         <dd>The base directory of an OCFL storage layout.</dd>
-        <dt><dfn>Version</dfn>:<dt>
-        <dd></dd>
+        <dt><dfn>OCFL Version</dfn>:<dt>
+        <dd>The state of an OCFL object's content which is constructed using the incremental
+          changes recorded in the sequence of corresponding and prior version directories.</a>.</dd>
```

Comment:
Travis is probably sad about the lonely `</a>`
https://travis-ci.com/OCFL/spec/builds/84563056#L466